skel/problem/generators/generators.yaml

```diff
@@ -66,19 +66,9 @@ data:
 
   invalid_inputs:
     # Add invalid testcase to ensure that input_validators correctly rejects them.
-    data:
-      empty:
-          in: ""
-      newline:
-          in: "\n"
-      random:
-          in: YVRtr&*teTsRjs8ZC2%kN*T63V@jJq!d
-      not_printable_1:
-          in: "\x7f"
-      not_printable_2:
-          in: "\xe2\x82\xac"
-      unicode:
-          in: "¯\\_(ツ)_/¯"
+    data: []
+      #dont_start_with_whitespace:
+      #    in: "<valid testcase>"
```

Comment:
Shouldn't there be an invalid test case here? Also, why is it called `dont_start_with_whitespace`?

Reply:
this is the same for every other subdir in the example yaml.

Reply:
Sure, but the `invalid_answer` is the only one that actually starts with whitespace :stuck_out_tongue: I've cleaned this section up a bit :slightly_smiling_face: